lbrynet/core/Wallet.py

```diff
@@ -313,12 +314,12 @@ def get_txid_for_name(self, name):
         d.addCallback(lambda r: None if 'txid' not in r else r['txid'])
         return d
 
-    def get_stream_info_from_txid(self, name, txid):
+    def get_stream_info_from_txid(self, name, txid, nout):
         d = self.get_claims_from_tx(txid)
 
         def get_claim_for_name(claims):
             for claim in claims:
-                if claim['name'] == name:
+                if claim['name'] == name and claim['nOut'] == nout:
```

Comment:
this logic gets repeated, and I generally dislike complicated if statements.  This might be better if there was a data structure, maybe:

```
ClaimLocation = collections.namedtuple('ClaimIdentifier', 'txid nout')
```

And then changed each function to take in a `claim_location` like

```
def get_stream_info_from_txid(self, name, claim_location):
```

and have a function:

```
def matchesClaim(claim, claim_location):
    return claim['name'] == claim_location.name and claim['nOut'] == claim_location.nout
```